api/src/com/cloud/network/vpc/OSPFZoneConfig.java

```diff
@@ -0,0 +1,332 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.cloud.network.vpc;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang.StringUtils;
+
+import com.cloud.exception.InvalidParameterValueException;
+import com.cloud.utils.net.NetUtils;
+
+public class OSPFZoneConfig {
+
+    private long zoneId;
+    private Protocol protocol;
+    private String ospfArea;
+    private Short helloInterval;
+    private Short deadInterval;
+    private Short retransmitInterval;
+    private Short transitDelay;
```

Comment:
@abhinandanprateek why did you chose to represent these time values as ``Short`` rather than ``Duration``?  My issue with ``Short`` is there is no notion of units (e.g. seconds, milliseconds) etc.  Therefore, the conversion semantics are impossible to determine when just represented as a numeric value.